frontend/src/components/Collections/components/Grid/components/Popover/index.tsx

```diff
@@ -6,13 +6,17 @@ import {
 } from "@blueprintjs/core";
 import { FC } from "react";
 import { LeftAlignedDetailsCell } from "../Row/common/style";
-import { ContentWrapper, FieldValues } from "./style";
+import { ContentColumn, ContentWrapper, FieldValues } from "./style";
 
 interface Props {
   values: string[];
 }
-
+const CHUNK_SIZE = 25;
 const Popover: FC<Props> = ({ values }) => {
+  const chunkedValues = Array(Math.ceil(values.length / CHUNK_SIZE))
```

Comment:
Looks good!

What do you think of this?

```ts
const chunk = (arr, size) =>
  Array.from({ length: Math.ceil(arr.length / size) }, (v, i) =>
    arr.slice(i * size, i * size + size)
  );
console.log(chunk([1, 2, 3, 4, 5], 2));
```

Got the idea from here 😆 

https://www.codegrepper.com/code-examples/javascript/divide+array+into+bucket+by+average